tools/travis/run-luacheck-windows.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+#Download luacheck binary if nessesary
+if ! [ -x "cache/luacheck.exe" ]; then
+    wget -O cache/luacheck.exe https://github.com/mpeterv/luacheck/releases/download/0.23.0/luacheck.exe
```

Comment:
I might make sense to define the retry params. You don't want a network hiccup to fail your build.